README.md

```diff
@@ -356,20 +357,20 @@ query = Emit(
    source
 )
 
-print json.dumps(query.to_jvalue(), indent=2)
+print(json.dumps(query.to_jvalue(), indent=2))
 
 results = query.eval(BuiltInEnv() | CommCareHqEnv(api_client) | JsonPathEnv())
 
 if len(list(env.emitted_tables())) > 0:
-    # with writers.Excel2007TableWriter("excel-output.xlsx") as writer:
-    with writers.StreamingMarkdownTableWriter(sys.stdout) as writer:
+    # with Excel2007TableWriter("excel-output.xlsx") as writer:
```

Comment:
nit: could also remove that comment I believe.

Reply:
a2f0bf6